btc-node-helper/index.js

```diff
@@ -88,14 +88,22 @@ module.exports = class BtcNodeHelper {
         return promisefy(this.client, this.client.generate, [blocks]);
     };
     
-    getBlock(blockHash) {
-        return promisefy(this.client, this.client.getBlock, [blockHash]);
+    getBlock(blockHash, deserialized = true) {
+        return promisefy(this.client, this.client.getBlock, [blockHash, deserialized]);
+    }
+
+    getBlockCount() {
+        return promisefy(this.client, this.client.getBlockCount, []);
     }
 
     getUtxos(address) {
         return promisefy(this.client, this.client.listUnspent, [null, null, [address]]);
     }
 
+    execute(cmd, args = []) {
+        return promisefy(this.client, this.client.cmd, [cmd].concat(args));
```

Comment:
If possible add these methods to the `sample/rpc-command.js`, so we can try it out and see how it works?

Reply:
I added a test.